azure-pipelines.yml

```diff
@@ -70,8 +70,8 @@ jobs:
   - script: |
       (
         cd src/mainboard/emulation/qemu-fsp
-        timeout 120s make run | tee serial
-        grep 'Running payload' serial
+        echo 'NO: timeout 120s make run | tee serial'
+        echo 'NO: grep Running.payload serial'
```

Comment:
The fsp job is just failing because of the fspsdk submodule change

Reply:
well what we could do ... just take this as is for now and start turning stuff back on.
I'd rather do that, as it might unblock a bunch of other PRs as well.

Reply:
Sounds good to me.

Reply:
Works for me. I'll re-enable this in a little bit